contents/blog/running-group-targeted-ab-tests.md

```diff
@@ -0,0 +1,93 @@
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-24
+featuredImage: ../images/blog/running-group-targeted-ab-tests/featured-image.png
+tags: ['experimentation']
+---
+
+A/B tests are a powerful tool for measuring how product changes impact user behavior. However, sometimes changing how one user interacts with your product will affect how others use it too. For example, a change in the Uber app for drivers likely affects riders' experience too. In these scenarios, group-targeted experiments enable us to measure the impact of changes beyond an individual user and across a group of users.
+
+## What are group-targeted experiments
+
+In group-targeted experiments, the test applies to an entire group rather than individual users. This means that everyone within a specific group will experience the same conditions, for example, the control or test variant. This enables you to measure the impact of any changes on the group as a whole, rather than on individual users within the group.
+
+Groups can be companies, geographic locations, or any other set of users with common characteristics.
+
+![Screenshot of setting the participant type in an experiment](../images/blog/running-group-targeted-ab-tests/user-vs-group-a-b-tests.png)
+
+## When to run group-targeted experiments
+
+Run group-targeted experiments when:
+
+* The change in how an individual user uses your product significantly affects the behavior of other users.
+
+* You want to measure the impact of a change on an entire group of users. 
+
+This is usually seen in the following types of products:
+
+1. B2B SaaS apps, in order to test how a change will affect how an entire company uses your product.
+
+   - For example, suppose Asana wants to test a new AI feature that automatically assigns tasks based on their urgency and importance. By conducting a group-targeted experiment, they can measure the impact on project completion rates across the entire company. 
+
+2. Products with network effects, since a change for one user will likely affect how other users interact with the product.
+
+   - For example, suppose Slack wants to improve the usage of a new video calling feature. Improving the feature's discoverability for a single user will increase their own usage with it, but since they use it with their coworkers, their coworkers will also discover it.
+
+## How does running a group-targeted experiment differ from a user-targeted one?
+
+There are three key differences to take into account when running a group-targeted experiment versus a user-level one:
+
+### 1. Less statiscal power
+
+Since you treat a group of users as a single data point in group-targeted experiments, you have less statistical power. For example, Slack has 20 million users, but only 600,000 companies using it.
+
+In practice, this means that you'll usually have to run group-targeted experiments longer than you would user-targeted experiments. It also means that your results may be more susceptible to noise.
```

Comment:
"More susceptible to noise" is a little unclear in this context, could use with some more clarification. Maybe another sentence on why noise is bad? It makes it less clear the success of an experiment?